packages/vxrn/src/create.ts

```diff
@@ -338,6 +341,33 @@ export const create = async (options: StartOptions) => {
           tsDecorators: true,
           mode: 'build',
         }),
+        {
+          name: 'assets',
+          async transform(_, id) {
+            const extension = extname(id)
+            const hash = createHash('md5').update(id).digest('hex')
+
+            const { width, height } = getImageSize(id)
+
+            if (new RegExp(`\\.(${SCALABLE_ASSETS.join('|')})$`).test(extension)) {
+              return `
+              import AssetRegistry from "react-native/Libraries/Image/AssetRegistry";
```

Comment:
That's one of the things I've spent too much time on 😅 Here we're importing React Native's internal and when doing so it is marked as `virtual:rn-internals:*`, and that's great because then all modules are exported in `@vxrn/react-native-prebuilt` and we could _potentially_ used 

https://github.com/natew/vxrn/blob/b5eb7afccaacaca42bb16751594d023f60a98f35/packages/react-native-prebuilt/prebuild-react-native.ts#L183-L196

But this magic code converts modules that are just re-exporting to direct source. Because `Libraries/Image/AssetRegistry` is just re-export of `@react-native/asset-registry`:

https://github.com/facebook/react-native/blob/main/packages/react-native/Libraries/Image/AssetRegistry.js

and that's why it fails to resolve it, when directly importing `@react-native/asset-registry/registry` it complains that it contains Flow types 🫠

Reply:
Woof yea things can get a bit complex. I can take a look tonight hopefully.

Some good news is were launching this product + hiring someone, so going forward a lot more time available.